eng/GetXCopyMSBuild.ps1

```diff
@@ -0,0 +1,37 @@
+# Lifted from https://github.com/dotnet/arcade/blob/main/eng/common/tools.ps1
```

Comment:
@MattGal this is pretty much what you suggested in dotnet/arcade#12402. Incorporating it in build.cmd was the only bit I added.

Reply:
Looks good to me.